### ISSUE TITLE
cmtk.bindir fails when cmtk is present in its build directory but not installed

### ISSUE THREAD

**Issue by jefferis** [jefferis/nat#378]
Assuming that you have built CMTK from source like https://www.nitrc.org/forum/message.php?msg_id=23835 and https://www.nitrc.org/forum/message.php?msg_id=23835 e.g.

```
cd /Users/jefferis/dev/neuroanat/cmtk/core/
mkdir build_3.3.1
cd build_3.3.1
ccmake ..
make
```
but not done
```
make install
```
and then added the `bin` directory in your build folder to your path 

```
> Sys.getenv('PATH')
[1] "/Users/jefferis/dev/neuroanat/cmtk/core/build_3.3.1/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin:/opt/X11/bin:/usr/local/git/bin:/usr/local/MacGPG2/bin:/Library/TeX/texbin:/opt/local/bin"
```
then cmtk.bindir will fail as follows:

```
> cmtk.bindir(set = T)
[1] "/Users/jefferis/dev/neuroanat/cmtk/core/build_3.3.1/lib/cmtk/bin"
> cmtk.version()
sh: /Users/jefferis/dev/neuroanat/cmtk/core/build_3.3.1/lib/cmtk/bin/dof2mat: No such file or directory
```

**Comment by jefferis** [comment]
@mmc46

**Comment by jefferis** [comment]
Closed by 911dcdd
